crates/iota-core/src/execution_cache/object_locks.rs

```diff
@@ -3,14 +3,15 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use dashmap::{DashMap, mapref::entry::Entry as DashMapEntry};
+use iota_common::*;
```

Comment:
Can we please import only the relevant things?

Reply:
Updated at [e243aec](https://github.com/iotaledger/iota/pull/4940/commits/e243aec9f2f4106d3a1b8b4cb09c290c65808a59).